File: 16676.cpp
#include <iostream>
#include <cstring>
#include <string>
using namespace std;

int main() {
	int N; cin >> N;
	string N_str = to_string(N);

	string make = "";
	for (int i = 0; i < N_str.length() - 1; i++)
		make += '1';
	make += '0';
	int cmp = atoi(make.c_str());

	if (N == 0) cout << 1 << endl;
	else if (N > cmp) cout << N_str.length() << endl;
	else cout << N_str.length() - 1 << endl;
}

File: 16676.py
N_str = input()
N = int(N_str)

make = ""
for i in range(len(N_str) - 1):
    make += '1'
make += '0'
cmp = int(make)

if N == 0: print(1)
elif N > cmp: print(len(N_str))
else: print(len(N_str) - 1)
